dpnp/tests/third_party/cupy/math_tests/test_misc.py

```diff
@@ -542,40 +542,39 @@ def test_convolve(self, xp, dtype):
         return xp.convolve(a, b, mode=self.mode)
 
 
-@pytest.mark.skip("convolve() is not implemented yet")
+# @pytest.mark.skip("convolve() is not implemented yet")
 @pytest.mark.parametrize("mode", ["valid", "same", "full"])
 class TestConvolve:
 
     @testing.for_all_dtypes(no_float16=True)
-    @testing.numpy_cupy_allclose(rtol=1e-6)
+    @testing.numpy_cupy_allclose(rtol=1e-6, type_check=has_support_aspect64())
```

Comment:
It seems passing for me locally without that change. Do we really need that?

Reply:
You are right.
It is needed only for `test_convolve_diff_types`